.github/workflows/build-and-test-linux-aarch64.yml

```diff
@@ -86,6 +86,10 @@ jobs:
               NUM_PROC_BUILD=$NUM_PROC_BUILD_MAX
             fi
 
+            [[ "${{ matrix.build-shared }}" == 'shared' ]] && BUILD_SHARED='ON' || BUILD_SHARED='OFF'
```

Comment:
With these short circuits, I'm reading this as `([[ "${{ matrix.build-shared }}" == 'shared' ]] && BUILD_SHARED='ON') || BUILD_SHARED='OFF'`, as otherwise there's no need to include `BUILD_SHARED`. Might consider adding braces (if that's an option).

Reply:
Being bash, there's plenty of ways to achieve the same effect: https://stackoverflow.com/questions/3953645/ternary-operator-in-bash. I'll leave this as is for now because it seems the no-braces is widely used.